src/handlebars/views/report.hbs

```diff
@@ -0,0 +1,10 @@
+<!-- FORM -->
+<div class="form-wrap">
+
+  <form class="report-form" action="/" method="post">
+
+    <button class="form-button" id="getLocationButton" type="button" name="location">Submit Location</button>
```

Comment:
probs best to stick to one naming convention - I think we've gone for kebab case in most places, so maybe change to get-location-button